assets/airports/eddf.json

```diff
@@ -6,940 +6,3986 @@
+                ["PSA", "CHA", "DF606", "DF610", "DF611", "DF612", "DF613", "DF614", "DF615", "DF616", "DF626", "DF625", "DF624", "DF623", "DF622", "DF621", "LEDKI"],
+                ["DF621", "REDGO"]
+            ]
         },
-        "PSA7S": {
-            "icao": "PSA7S",
-            "name": "Spessart Seven South",
+        "ROLIS": {
+            "icao": "ROLIS",
+            "name": "Rolis",
+            "suffix": {"07L":"07N", "07C":"07S", "07R":"07S", "25L":"25S", "25C":"25S", "25R":"25N"},
             "entryPoints": {
-                "KOVAN": ["KOVAN", ["ETASA", "A260-"], ["XINLA", "A240-"], ["SUKON", "A160-"], ["SUPIX", "A140-"]]
+                "ROLIS": ["ROLIS"]
             },
-            "body": [["PSA", "A110|S250"], "CHA", "DF635", "DF636", "DF640", "DF641", "DF642", "DF643", "DF644", "DF654", "DF653", "DF652", "DF651", "ROBSA"],
+            "body": [],
             "rwy": {
-                "07L": [],
-                "07C": [],
-                "07R": []
+                "07L": ["ETARU", "DF432", "DF439", "DF441", "DF442", "DF443", "DF444", "DF454", "DF453", "DF452"],
+                "07C": ["OSPUL", "DF632", "FFM", "DF621", "DF622", "DF623", "DF613", "DF612", "DF611", "DF635", "DF636", "DF640", "DF641", "DF642", "DF643", "DF644", "DF654", "DF653", "DF652", "DF651", "LOMPO"],
+                "07R": ["OSPUL", "DF632", "FFM", "DF621", "DF622", "DF623", "DF613", "DF612", "DF611", "DF635", "DF636", "DF640", "DF641", "DF642", "DF643", "DF644", "DF654", "DF653", "DF652", "DF651", "ROBSA"],
+                "25L": ["OSPUL", "DF402", "DF601", "DF609", "DF610", "DF611", "DF612", "DF613", "DF614", "DF615", "DF616", "DF626", "DF625", "DF624", "DF623", "DF622", "DF621", "LEDKI"],
+                "25C": ["OSPUL", "DF402", "DF601", "DF609", "DF610", "DF611", "DF612", "DF613", "DF614", "DF615", "DF616", "DF626", "DF625", "DF624", "DF623", "DF622", "DF621", "REDGO"],
+                "25R": ["ETARU", "DF403", "DF409", "DF410", "DF411", "DF412", "DF413", "DF414", "DF415", "DF416", "DF426", "DF425", "DF424", "DF423", "DF422", "DF421"]
             },
-            "draw": [[]]
+            "draw": [
+                ["ROLIS", "ETARU", "DF432", "DF439", "DF441", "DF442", "DF443", "DF444", "DF454", "DF453", "DF452"],
+                ["ETARU", "DF403", "DF409", "DF410", "DF411", "DF412", "DF413", "DF414", "DF415", "DF416", "DF426", "DF425", "DF424", "DF423", "DF422", "DF421"],
+                ["ROLIS", "OSPUL", "DF632", "FFM", "DF621", "DF622", "DF623", "DF613", "DF612", "DF611", "DF635", "DF636", "DF640", "DF641", "DF642", "DF643", "DF644", "DF654", "DF653", "DF652", "DF651", "LOMPO"],
+                ["DF651", "ROBSA"],
+                ["OSPUL", "DF402", "DF601", "DF609", "DF610", "DF611", "DF612", "DF613", "DF614", "DF615", "DF616", "DF626", "DF625", "DF624", "DF623", "DF622", "D
```

Comment:
The video map is still quite crowded :thinking: At this point I think we should get rid of the top-down view of EDDF, even though I think it looks really cool, it's very hard to tell where the runways actually are and that's all a radar controller really cares about. The symbols for `VFM` and `FFM` can be removes as well since they're not super important (not anymore anyway). You can also remove those airspace division lines, they seem to be MVA's which aren't essential

Reply:
I really don't think it is crowded, but I guess I'll have to :/

Reply:
You see, I think the videomap looks pretty empty after removing all that...

Reply:
It's not too bad now, so you can leave it as is

Reply:
Thank you @Fechulo